### ISSUE TITLE
Presence of "no name" item

### ISSUE THREAD

**Issue by firthm01** [ebu/ear-production-suite#28]
See attached REAPER project. The Scene shows an item with "no name" label. Needs investigation to discover why this is happening.
[Template.zip](https://github.com/ebu/ear-production-suite/files/5688239/Template.zip)

**Comment by rsjtaylor** [comment]
This might be related to what you were talking about @chrispike-bbc - If you add an issue could you tag this one please?

**Comment by chrispike-bbc** [comment]
Yes, that's it! I've seen this several times. I haven't yet worked out how to recreate it though.

**Comment by ial-sebastian** [comment]
When i try to load the attached project, it fails with a message that it couldn't find the VSTs. I guess this happens because some GUID or checksum has changed. When i manually replace the plugins and save the project, it probably fixes itself in the process, because i'm not able to see the issue afterwards. Is there any "recipe" you followed in order to create this project, so i could try that out?

**Comment by chrispike-bbc** [comment]
Sure. 

- Create a project using the ADM file in the attached zipped folder. 
- Then delete one of the EAR Object plug-ins from the created tracks
- `no name` appears in the EAR Scene plug-in

[HaydnCoaches.wav.zip](https://github.com/ebu/ear-production-suite/files/5952100/HaydnCoaches.wav.zip)

**Comment by ial-sebastian** [comment]
Thanks, with this i'm able to reproduce it.

**Comment by firthm01** [comment]
> When i try to load the attached project, it fails with a message that it couldn't find the VSTs. I guess this happens because some GUID or checksum has changed. 

Hi Sebastian - you're right. Originally we used the same codes for the entire suite since it didn't seem to make any difference. However, it turned out some DAWs didn't like it, so we recently gave them all unique codes. If you run in to this issue again, there's a conversion tool in the `tools` directory of the repository which should be able to update the codes stored in an existing RPP file.
